Dockerfile

```diff
@@ -1,37 +1,45 @@
-FROM ruby:3.1.2
-RUN apt-get update -qq && apt-get upgrade -y && apt-get install -y build-essential \
-  libpq-dev libxml2-dev libxslt1-dev dumb-init default-jre
+ARG base_image=ghcr.io/alphagov/govuk-ruby-base:3.1.2
+ARG builder_image=ghcr.io/alphagov/govuk-ruby-builder:3.1.2
 
-ENV APP_HOME /smokey
-RUN mkdir $APP_HOME
-
-ENV APT_KEY_DONT_WARN_ON_DANGEROUS_USAGE 1
-
-# Install Google Chrome
-RUN wget -q -O - https://dl-ssl.google.com/linux/linux_signing_key.pub | apt-key add -
-RUN sh -c 'echo "deb [arch=amd64] http://dl.google.com/linux/chrome/deb/ stable main" >> /etc/apt/sources.list.d/google.list'
-RUN apt-get update && apt-get install -y google-chrome-stable
+FROM $builder_image AS builder
 
 WORKDIR $APP_HOME
-ADD Gemfile* $APP_HOME/
-ADD .ruby-version $APP_HOME/
+COPY Gemfile* .ruby-version ./
 RUN bundle install
+COPY . ./
+
+
+FROM $base_image
+SHELL ["/bin/bash", "-o", "pipefail", "-c"]
+
+# Remove Cucumber advert.
+ENV CUCUMBER_PUBLISH_QUIET=true
+
+# Install Google Chrome and the corresponding version of ChromeDriver.
+ARG chromedriver_url=https://chromedriver.storage.googleapis.com/
+ADD https://dl-ssl.google.com/linux/linux_signing_key.pub /tmp/google.pub
+RUN arch=$(dpkg --print-architecture) && \
+    keyring=/usr/share/keyrings/google-linux-signing.gpg && \
+    gpg --dearmor < /tmp/google.pub > "${keyring}" && \
+    echo "deb [arch=${arch} signed-by=${keyring}] https://dl.google.com/linux/chrome/deb/ stable main" \
+        > /etc/apt/sources.list.d/google.list && \
+    install_packages dumb-init google-chrome-stable unzip && \
+    rm -fr /tmp/*
+RUN chrome_ver=$(google-chrome --version | grep -Po '\d+\.\d+\.\d+') && \
+    chromedriver_ver=$(curl -LSfs "${chromedriver_url}LATEST_RELEASE_${chrome_ver}") && \
+    curl -LSfs "${chromedriver_url}${chromedriver_ver}/chromedriver_linux64.zip" \
+        | funzip >/usr/bin/chromedriver && \
+    chmod 755 /usr/bin/chromedriver
```

Comment:
I think _all of this gumpf_ is the reason why I originally switched over to Chromium rather than Google Chrome 😅 

Having Chromium in the apt package manager made it as simple as `apt install chromium chromium-driver`, which felt like a much less noisy way to do things.

However: this is a necessary evil if we want/need to be testing against Google Chrome 👍🏻

Reply:
Yeah 💯 — tbh I would have just switched it back to Chromium had I not seen that we'd only recently switch to Chrome in #1049 😅 Certainly took me a while to script that ChromeDriver installation, but hopefully it's a more robust outcome overall.

Reply:
I suspect we may have to switch back at some point in order to support arm64, but with any luck this should tide us over until everything's on k8s.